tests/test_add_to_wishlist.py

```diff
@@ -28,17 +26,17 @@ def test_add_to_wishlist(self, driver):
 
         # Find and click on the 'Apparel & Shoes' link
         with allure.step("Find and click on the 'Apparel & Shoes' link and add a product to the wishlist"):
-            self.wait_for_element_and_click(driver, WishlistPageLocators.APPAREL_SHOES_LINK, 20)
```

Comment:
here and in future all tests
you have class with locators WishlistPageLocators, also you can create WishlistPageMethods
where you can add separate methods as:
@staticmethod
def click_on_apparel_shoes_link(driver):
self.wait_for_element_and_click(driver, WishlistPageLocators.APPAREL_SHOES_LINK, 20)


and use this method as 
WishlistPageMethods.click_on_apparel_shoes_link(driver)
which would be better for understanding purpose of tests

same for another methods